src/components/Button/Button.tsx

```diff
@@ -41,7 +46,7 @@ export const Button = ({
         }
       }}
       tabIndex={tabIndex}
-      type={!href ? type : null}
+      type={!href ? type : 'button'}
```

Comment:
This needs to be left as `type={!href ? type : null}`

If an href is specified the component is rendered as an <a> as opposed to a <button>

modify line 28 instead to be `type = 'button',`

Reply:
🤦🏻‍♂️ of course! I've misread the conditional - will update

Reply:
fixed